python/lsst/afw/math/chebyshevBoundedField.cc

```diff
@@ -21,16 +21,72 @@
  */
 
 #include <pybind11/pybind11.h>
-//#include <pybind11/operators.h>
-//#include <pybind11/stl.h>
+#include <pybind11/operators.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/afw/math/BoundedField.h"
+#include "lsst/afw/math/ChebyshevBoundedField.h"
 
 namespace py = pybind11;
+PYBIND11_DECLARE_HOLDER_TYPE(MyType, std::shared_ptr<MyType>);
 
 using namespace lsst::afw::math;
 
 PYBIND11_PLUGIN(_chebyshevBoundedField) {
     py::module mod("_chebyshevBoundedField", "Python wrapper for afw _chebyshevBoundedField library");
+    
+    if (_import_array() < 0) {
+            PyErr_SetString(PyExc_ImportError, "numpy.core.multiarray failed to import");
+            return nullptr;
+    };
 
+    /* ChebyshevBoundedFieldControl */
+    py::class_<ChebyshevBoundedFieldControl>
+        clsChebyshevBoundedFieldControl(mod, "ChebyshevBoundedFieldControl");
+    /* ChebyshevBoundedFieldControl  Constructors */
+    clsChebyshevBoundedFieldControl.def(py::init<>());
+    /* ChebyshevBoundedFieldControl Members */
+    clsChebyshevBoundedFieldControl.def_readwrite("orderX", &ChebyshevBoundedFieldControl::orderX);
+    clsChebyshevBoundedFieldControl.def_readwrite("orderY", &ChebyshevBoundedFieldControl::orderY);
+    clsChebyshevBoundedFieldControl.def_readwrite("triangular", &ChebyshevBoundedFieldControl::triangular);
+    
+    typedef ChebyshevBoundedFieldControl Control;
+    
+    /* ChebyshevBoundedFieldControl */
+    py::class_<ChebyshevBoundedField, std::shared_ptr<ChebyshevBoundedField>, BoundedField>
+        clsChebyshevBoundedField(mod, "ChebyshevBoundedField");
+    /* ChebyshevBoundedFieldControl  Constructors */
+    clsChebyshevBoundedField.def(py::init<lsst::afw::geom::Box2I const &, ndarray::Array<double const,2,2>
+        const &>());
+    /* ChebyshevBoundedFieldControl  Operators */
+    clsChebyshevBoundedField.def("__rmul__", [](ChebyshevBoundedField &bf, double const scale){
+            return bf*scale;
+    }, py::is_operator());
+    clsChebyshevBoundedField.def("__mul__", [](ChebyshevBoundedField &bf, double const scale){
+            return bf*scale;
+    }, py::is_operator());
```

Comment:
In this case we could probably use `.def(py::self += double())` too (with the `pybind11/operators.h` header). But I'm not yet convinced about the nicest strategy for wrapping operators. We need to talk about this at some point. Until then, this is fine.